packages/twenty-front/src/pages/settings/roles/components/RoleAssignmentTableRow.tsx

```diff
@@ -0,0 +1,98 @@
+import { Table } from '@/ui/layout/table/components/Table';
+import { TableCell } from '@/ui/layout/table/components/TableCell';
+import { TableRow } from '@/ui/layout/table/components/TableRow';
+import styled from '@emotion/styled';
+import { t } from '@lingui/core/macro';
+import {
+  AppTooltip,
+  Avatar,
+  IconButton,
+  IconTrash,
+  TooltipDelay,
+} from 'twenty-ui';
+import { WorkspaceMember } from '~/generated-metadata/graphql';
+
+const StyledTable = styled(Table)`
+  margin-top: ${({ theme }) => theme.spacing(0.5)};
+`;
+
+const StyledIconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin-right: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledTextContainerWithEllipsis = styled.div`
+  overflow: hidden;
```

Comment:
I think we already have this kind of compponent that will automatically add the tooltip, we should re-use components as much as we can!